README.md

```diff
@@ -472,8 +477,60 @@ Create `.eslintrc` at root of your project that contains:
 ```json
 {
   "root": true,
-  "extends": [
-    "@jupiterone/eslint-config/react"
-  ]
+  "extends": ["@jupiterone/eslint-config/react"]
 }
 ```
+
+## Usage: Husky 7
+
+In the updates to Husky for major version 7 they have included some large
```

Comment:
Can you add the recommended contents of `.husky/pre-commit` to this section?

Basically, I want developers to know that they need to create `.husky/pre-commit` with this content:

```sh
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npx lint-staged
```

Do you know if you need ot make the `pre-commit` file executable?

Reply:
On this-- there's a husky tool for this express purpose that creates the file correctly.